helm_deploy/values-prod.yaml

```diff
@@ -17,6 +17,7 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_SERVICE_NAME: approved-premises
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    NEW_WITHDRAWLS_FLOW_DISABLED: 1
```

Comment:
I'm happy to be wrong here, but is it not 'withdrawal'?

Reply:
Oh man, I struggle so much with that word!